AI/BattleAI/AttackPossibility.h

```diff
@@ -36,6 +38,12 @@ class AttackPossibility
 
 	static AttackPossibility evaluate(const BattleAttackInfo & attackInfo, BattleHex hex, const HypotheticBattle * state);
 
+	static int64_t calculateDpsReduce(
```

Comment:
Please add decryption of Dps in the comment

Reply:
not quite sure why `dps` term is used - simple `damage` would do

Reply:
I think damage is better here)